doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-commands-samples-ref.adoc

```diff
@@ -0,0 +1,876 @@
+String payload = marshaller.marshall(batch);
+----
+
+.Example REST request body (JSON)
+[source,json]
+----
+{
+  "commands" : [ {
+    "call-container" : {
+      "payload" : "{\n  \"lookup\" : \"defaultKieSession\",\n  \"commands\" : [ {\n    \"fire-all-rules\" : {\n      \"max\" : -1,\n      \"out-identifier\" : null\n    }\n  } ]\n}",
+      "container-id" : "command-script-container"
+    }
+  } ]
+}
+----
+
+.Example server response (JSON)
+[source,json]
+----
+{
+  "response": [
+    {
+      "type": "SUCCESS",
+      "msg": "Container employee-rostering successfully called.",
+      "result": "{\n  \"results\" : [ ],\n  \"facts\" : [ ]\n}"
+    }
+  ]
+}
+----
+--
+DisposeContainerCommand::
++
+--
+Disposes a specified KIE container in the {KIE_SERVER}.
+
+.Command attributes
+[cols="30%,50%,20%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Requirement
+
+|`containerId`
+|ID of the KIE container to be disposed
+|Required
+
+|
+|
+|Optional
+|===
+
+.Example Java command
+[source,java]
+----
+
+----
+
+.Example REST request body (JSON)
+[source,json]
+----
+{
+  "commands" : [ {
+    "dispose-container" : {
+      "container-id" : "MyContainer"
+    }
+  } ]
+}
+----
+
+.Example server response (JSON)
+[source,json]
+----
+{
+  "response": [
+    {
+      "type": "SUCCESS",
+      "msg": "Container MyContainer successfully disposed.",
+      "result": null
+    }
+  ]
+}
+----
+--
+GetScannerInfoCommand::
++
+--
+Returns information about the KIE scanner used for automatic updates in a specified KIE container, if applicable.
+
+.Command attributes
+[cols="30%,50%,20%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Requirement
+
+|`containerId`
+|ID of the KIE container where the KIE scanner is used
+|Required
+
+|
+|
+|Optional
+|===
+
+.Example Java command
+[source,java]
+----
+
+----
+
+.Example REST request body (JSON)
+[source,json]
+----
+{
+  "commands" : [ {
+    "get-scanner-info" : {
+      "container-id" : "command-script-container"
+    }
+  } ]
+}
+----
+
+.Example server response (JSON)
+[source,json]
+----
+{
+  "response": [
+    {
+      "type": "SUCCESS",
+      "msg": "Scanner info successfully retrieved",
+      "result": {
+        "kie-scanner": {
+          "status": "DISPOSED",
+          "poll-interval": null
+        }
+      }
+    }
+  ]
+}
+----
+--
+UpdateScannerCommand::
++
+--
+Starts or stops a KIE scanner that controls polling for updated KIE container deployments.
+
+.Command attributes
+[cols="30%,50%,20%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Requirement
+
+|`containerId`
+|ID of the KIE container where the KIE scanner is used
+|Required
+
+|`status`
+|Status to be set on the new KIE container (`STARTED`, `STOPPED`)
+|Required
+
+|`pollInterval`
+|Permitted polling during in milliseconds
+|Required only when starting scanner
+
+|
+|
+|Optional
+|===
+
+.Example Java command
+[source,java]
+----
+
```

Comment:
all missing client commands are simple methods of KieServicesClient

Reply:
Done.